doc/articles/platform-specific-xaml.md

```diff
@@ -133,3 +133,10 @@ More visually, platform support for the pre-defined prefixes is shown in the bel
 Where:
  * 'Win' represents Windows, and
  * 'Droid' represents Android
+
+### Cross target library
+Since version 3.0, the Uno platform prefixes have changed for cross targeted libraries. These libraries need to be compiled for every possible platform. For most platform, that does not change anything, but with the addition of Skia, the Uno platform cannot distinguish Skia and Wasm. 
+
+The prefix `netstdref` is available and will include the objects or properties in both Skia and Wasm build. A prefix `not_nestdref` can also be used to exclude them. Since Skia and Wasm are similar, it is often not necessary to make the distinction. 
+
+In cases where it is needed (fonts are one example) then the Xaml files must be placed directly in the respective head or in a shared library.
```

Comment:
What do you mean by 'in a shared library' here?

Reply:
I was refering to the shared project, Il change the sentence to

In cases where it is needed (fonts are one example) then the XAML files must be placed directly in the platform specific project or a shared project.